internal/healthchecks/api_check.go

```diff
@@ -108,7 +108,7 @@ func runLoggingCheck(logger logs.StructuredLogger) error {
 	logger.Infof("logging client was created successfully")
 
 	if err := logClient.Ping(ctx); err != nil {
-		logger.Println(err)
+		logger.Errorf(err.Error())
```

Comment:
Here we are producing an log with severity=ERROR that doesn't have a code. Is that something we really want? We can just warn here or infof since we don't have a code for this error.